scripts/bosh-scp.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash 
+
+set -euo pipefail
+
+function run_command() {
+    local command=$1
+    local message=$2
+
+    echo -n "${message}... "
+
+    if ! output=$(bash -c "${command}" 2>&1); then
+        echo -e "failed.\n\n"
+        echo "Command Failed: ${command}"
+        echo ""
+        echo "Output: ${output}"
+        exit 1
+    fi
+    echo "done."
+}
+
+command -v jq >/dev/null 2>&1 || { echo >&2 "jq is required but it's not installed.  Aborting."; exit 1; }
+command -v bosh >/dev/null 2>&1 || { echo >&2 "bosh is required but it's not installed.  Aborting."; exit 1; }
+command -v cut >/dev/null 2>&1 || { echo >&2 "cut is required but it's not installed.  Aborting."; exit 1; }
+
+bosh vms --json | jq -r '.Tables[0].Rows[] .instance' | while read -r instance; do
```

Comment:
You could make this a little simpler by doing the filtering in JQ itself

```
bosh vms --json | jq '.Tables[0].Rows[] | select(.instance|startswith("rds_broker/"))'
```

Reply:
updated